### PR TITLE
4 get reviews

### PR DESCRIPTION
**Pull request by wisemanrick**
Completed ticket 4 with a custom test for to check if the data is sorted correctly

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -33,10 +33,34 @@ describe("GET /api/categories", () =>{
...
+
+describe("GET /api/reviews", () =>{
+    test("Should have a length of 4 and 2 properties 'slug' and 'description'", () => {
+        return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({ body }) => {
+            const { reviews } = body
+            expect(reviews).toHaveLength(13)
+            //console.log(reviews)
```
Comment (yalicee) [inline __tests__/app.test.js]: delelte logs pls

**Review by yalicee — APPROVED** [review]
Looks great :)

**Comment by wisemanrick** [comment]
issues with package JSON

File: __tests__/app.test.js [hunk 2]
```diff
@@ -26,17 +27,68 @@ describe("GET /api/categories", () =>{
...
+        .expect(200)
+        .then(({ body }) => {
+            const { reviews } = body
+            const constclonedReviews = [...reviews]
+            const sortedByReview_id = constclonedReviews.sort((sortA, sortB) => {
+                return sortB.created_at - sortA.created_at
+            })                 
+            expect(reviews[0].created_at).toBe(sortedByReview_id[0].created_at)
+            expect(reviews[12].created_at).toBe(sortedByReview_id[12].created_at)
+        })
```
Comment (Beth-McCormack) [inline __tests__/app.test.js:74]: This is expecting the reviews to be in the same order before and after they're sorted. Have a look at the jest-sorted library and see if you can implement that here instead.

**Review by Beth-McCormack — CHANGES_REQUESTED** [review]
see comments

**Pull request merged**